### PR TITLE
Rename --quiet flag to --no-progressbar

### PR DESCRIPTION
**Pull request by vadmeste**
*) '--quiet' claims that it is generating less chatting messages but it doesn't, 'mc ls' can fill many pages of my screen
*) '--quiet is little vague, the user cannot speculate its effect on the command
*) --no-progressbar is much more specific and the user could easily know its effect

### CODE REVIEW

**Comment by harshavardhana** [comment]
> vadmeste commented 2 hours ago
> *) '--quiet' claims that it is generating less chatting messages but it doesn't, 'mc ls' can fill many pages of my screen
> *) '--quiet is little vague, the user cannot speculate its effect on the command
> *) --no-progressbar is much more specific and the user could easily know its effect

The problem here is 

```
mc ls --no-progressbar
``` 

Has no real meaning, in that sense '--quiet' is a better choice since it's more generic.

**Comment by vadmeste** [comment]
yes, I understand that but I think that the user won't invoke this flag since he already knows that it does nothing in the case of ls, at the same time --quiet is not quiet at all with ls..

**Comment by harshavardhana** [comment]
> yes, I understand that but I think that the user won't invoke this flag since he already knows that it does nothing in the case of ls, at the same time --quiet is not quiet at all with ls..

Then it shouldn't be a global option, '--quiet' should be moved into only '--no-progressbar'  for cp and mirror.

File: session-v7.go [hunk 1]
```diff
@@ -32,14 +32,14 @@ import (
 	"github.com/minio/minio/pkg/quick"
 )
 
-func migrateSessionV5ToV6() {
+func migrateSessionV6ToV7() {
```
Comment (harshavardhana) [inline session-v7.go]: Another thing is we cannot just do migrate and forget older v6 versions, we need to add migration for v5-->v6 and then v6-->v7 - just like how we do for config files.

**Comment by abperiasamy** [comment]
Reason to name it as --quiet is keep it generic. It would converge multiple related options into one, such as --no-progress-bar.

File: session-v7.go (hunk 1 above)
Comment (vadmeste) [inline session-v7.go]: yes, I saw you did that with config migration, however in session, migration does nothing more than deleting old session files, this code  will clean all versions inferior to 7

Comment (harshavardhana) [inline session-v7.go]: > yes, I saw you did that with config migration, however in session, migration does nothing more than deleting old session files, this code will clean all versions inferior to 7

It shouldn't be it was only for V5 --> V6, but V6 onwards it needs to migrate.

Comment (vadmeste) [inline session-v7.go]: but I know that in general, sequential migration is a good idea to avoid bugs and hidden problems

Comment (vadmeste) [inline session-v7.go]: oh okay, are you saying that i should replace quiet with no progress bar ?

Comment (vadmeste) [inline session-v7.go]: I mean in session migration

**Comment by vadmeste** [comment]
yes but still, mc --quiet mirror is really chatty for the current mc.

What about doing the thing in reverse ? Making mc quiet by default but it uses the progress bar, and if the user wants to see more details, he can invoke  --chatty for example, then we will not need --quiet or --no-progressbar

**Pull request closed without merging**